Sources/Script/DependencyName.swift

```diff
@@ -46,7 +46,7 @@ extension ImportSpecification.DependencyName: Codable {
                 case .path(let path):
                     localRelativePathPrefix = path.parent
                 case .string:
-                    localRelativePathPrefix = Path.cwd
+                    localRelativePathPrefix = Path(Path.cwd)
```

Comment:
Not sure if this cast is correct, but was part of the `Path.swift` upgrade.

Reply:
Yeah it's necessary until Swift has existentials.